@xen-orchestra/rest-api/package.json

```diff
@@ -28,7 +28,10 @@
     "typescript-eslint": "^8.23.0"
   },
   "dependencies": {
+    "@vates/types": "^0.0.0",
     "@xen-orchestra/log": "^0.7.1",
+    "inversify": "^6.2.2",
```

Comment:
is this new dependecy mandatory ? could you explain the main benefits ?

Reply:
Yes. `inversify` is the dependency that makes DI possible in this PR.
BTW, `inversify-binding-decorators`, is not mandatory, but is a helper that provides tools to not declare for each class how to inject dependencies